modules/lvms-about-scaling-storage-of-clusters.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="lvms-about-scaling-storage-of-cluster_{context}"]
+= About scaling up the storage of clusters
+
+{product-title} supports additional worker nodes for clusters on bare-metal user-provisioned infrastructure. {lvms} detects and uses additional worker nodes when the nodes become active.
```

Comment:
These are two different options to scale the storage of a cluster: 
1. Adding additional worker nodes with available storage
2. Adding new devices to already existing worker nodes

Should we make this more obvious?